medical/tests/test_medical_patient.py

```diff
@@ -139,3 +142,21 @@ def test_check_birthdate_date(self):
                     now + timedelta(days=20),
                 )
             })
+
+    def test_search_age(self):
+        """
+        When patients are searched by age,
+        it should return patients with the corresponding birth dates
+        """
+        birthdate = datetime.strptime(
+            self.patient_1.birthdate, "%Y-%m-%d"
+        ).date()
+        current_date = date.today()
+        delta = current_date - birthdate
+        years = delta.days/365 - 1
+        result = self.env['medical.patient'].search(
+            [('age', 'like', years)]
```

Comment:
Out of curiosity, where is the operator coming from when the user does a search? Is there a default operator set up in Odoo? I'm a little concerned about using `=` when the age is listed as X yrs X mos X days (not just an integer of yrs).